File: ejercicio2.cpp
#include <iostream>
#include <vector>
#include <cstdlib>
#include <ctime>

using namespace std;

int main() {
    vector<int> valores;
    srand(time(0));

    for (int i = 0; i < 500; i++) {
        valores.push_back(rand() % 51 + 50);
    }

    vector<int> valoresPares;
    vector<int> valoresImpares;

    for (int val : valores) {
        if (val % 2 == 0) {
            valoresPares.push_back(val);
        } else {
            valoresImpares.push_back(val);
        }
    }

    cout << "Valores Pares:" << endl;
    for (int val : valoresPares) {
        cout << val << " ";
    }
    cout << endl << endl;

    cout << "Valores Impares:" << endl;
    for (int val : valoresImpares) {
        cout << val << " ";
    }
    cout << endl;

    return 0;
}

File: Ejercicio2.py
import random as rd
valores = []
for i in range(500):
    valores.append(rd.randint(50,100))

valoresPares = []
valoresImpares = []

for i in valores:
    if i%2 == 0:
        valoresPares.append(i)
    else:
        valoresImpares.append(i)
        
print("Valores Pares")
print(valoresPares)
print("Valores Impares")
print(valoresImpares)